pennylane/io.py

```diff
@@ -410,6 +410,62 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    .. note::
+
+        This function depends upon the PennyLane-Qiskit plugin. Follow the
+        `installation instructions <https://docs.pennylane.ai/projects/qiskit/en/latest/installation.html>`__
+        to get up and running. You may need to restart your kernel if you are running in a notebook
+        environment.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a ``NoiseModel`` constructed in Qiskit
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        kraus_shape (bool): use shape of the Kraus operators to display ``qml.QubitChannel``
+            instead of the complete list of matrices. Default is ``True``.
+        decimal_places (int): number of decimal places to round the Kraus matrices. Default is ``10``.
```

Comment:
Do we need this?

Reply:
Allows you to control rounding for matrices when verbose is ``False``
```py
>>> import pennylane_qiskit
>>> import qiskit_aer.noise as noise
>>> error_1 = noise.depolarizing_error(0.001, 1)
>>> error_2 = noise.depolarizing_error(0.01, 2)
>>> noise_model = noise.NoiseModel()
>>> noise_model.add_all_qubit_quantum_error(error_1, ['rz', 'ry'])
>>> noise_model.add_all_qubit_quantum_error(error_2, ['cx'])
>>> pennylane_qiskit.load_noise_model(noise_model, verbose=False, decimal_places=4)
NoiseModel({
    OpIn(['RZ', 'RY']): QubitChannel(Klist=[array([[-0.9996+0.j,  0.    +0.j],
       [ 0.    +0.j, -0.9996+0.j]]), array([[-0.0158+0.j,  0.    +0.j],
       [ 0.    +0.j,  0.0158+0.j]]), array([[0.    +0.j, 0.    +0.j],
       [0.0224+0.j, 0.    +0.j]]), array([[0.    +0.j, 0.0224+0.j],
       [0.    +0.j, 0.    +0.j]])])
    OpIn(['CNOT']): QubitChannel(Klist=[array([[-0.0433+0.j,  0.    +0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j,  0.0144+0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.0144+0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.    +0.j,  0.0144+0.j]]), array([[-0.9953+0.j,  0.    +0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j, -0.9953-0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j, -0.9953-0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.    +0.j, -0.9953+0.j]]), array([[ 0.    +0.j,  0.    +0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j, -0.0204+0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j, -0.0204+0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.    +0.j,  0.0408+0.j]]), array([[ 0.    +0.j,  0.    +0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j, -0.0354+0.j,  0.    +0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.0354+0.j,  0.    +0.j],
       [ 0.    +0.j,  0.    +0.j,  0.    +0.j, -0.    +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.05+0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.05+0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.05+0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.05+0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.05+0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.05+0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.05+0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.05+0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.05+0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.05+0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.05+0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]]), array([[0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.05+0.j],
       [0.  +0.j, 0.  +0.j, 0.  +0.j, 0.  +0.j]])])
})
```